index.md

```diff
@@ -0,0 +1,54 @@
+--- 
+layout: default 
+--- 
+
+## rdiff-backup is reverse differential backup ## 
+rdiff-backup backs up one directory to another, possibly over a network. The target directory ends up a copy of the source directory, but extra reverse diffs are stored in a special subdirectory of that target directory, so you can still recover files lost some time ago. The idea is to combine **the best features of a mirror and an incremental backup**. rdiff-backup also preserves subdirectories, hard links, dev files, permissions, uid/gid ownership (if it is running as root), modification times, extended attributes, acls, resource forks etc. Finally rdiff-backup can operate in a **bandwidth efficient** manner over a pipe, like rsync. Thus you can use rdiff-backup and ssh to securely back a hard drive up to a remote location, and only the differences will be transmitted. Finally, rdiff-backup is **easy to use** and settings have sensical defaults. 
+
+[Sol1][sol1gh] has officially taken over stewardship of rdiff-backup from February 2016. [Sol1][sol1gh] has long been a contributor and user of rdiff-backup, and will maintain the open source nature of rdiff-backup, while bringing it into the modern era.
+
+## Usage ##
+
+The wiki contains documentation on use cases, and serves as a respository of scripts for people to contribute to. These scripts do many things for and around rdiff-backup.
+
+## Authors and Contributors ## 
```

Comment:
there's overlap with content on index.md and acknowledgements.md
should we put this info in the index or elsewhere?